### ISSUE TITLE
[1.9.4] Implement new IFluidHandler capabilities so Forestry farms and others work again

### ISSUE THREAD

**Issue by RealGrep** [SleepyTrousers/EnderIO#3375]
#### Issue Description:
Fluid conduits do not attach to Forestry farms. They implemented the new IFluidHandler capabilities, and fluid conduits no longer work with them, or presumably other mods that use the new functionality. Figured it would be helpful if I logged a PR to highlight the issue.

#### What happens:
Fluid conduit will not attach to Forestry farm.

#### What you expected to happen:
That the conduit attach and function with the Forestry farm.

#### Steps to reproduce:

1. Get the latest Forestry.
2. Make farm
3. Try and attach conduit to Farm Valve. It will not. Fluid tank or reservoir will not be able to fill it, either.

____
#### Affected Versions (Do *not* use "latest"):

- EnderIO: EnderIO-1.9.4-3.0.0.37_alpha.jar
- EnderCore: EnderCore-1.9.4-0.4.0.23-alpha.jar
- Minecraft: 1.9.4
- Forge: 1.9.4-12.17.0.1954

#### Your most recent log file where the issue was present: 

No errors or other relevant information appears in the log.

**Comment by tkdberger** [comment]
>EnderIO: EnderIO-1.9.4-3.0.0.37_alpha.jar
So how exactly did you get to using a fluid conduit when you can't even look though the item list? Did you manually craft it, or do something extra cheeky? 
Currently I can't even open the creative menu tab for the mod, let alone test this

**Comment by RealGrep** [comment]
Yep, crafted it normally.

**Comment by Rumpelstiltskinny** [comment]
Yeah, support for IFluidHandler would be appreciated.

**Comment by mezz** [comment]
Since this is not a priority, I was able to restore compatibility in Forestry by implementing the old  IFluidHandler interface in addition to the capabilities.

**Comment by RealGrep** [comment]
ActuallyAdditions also reimplemented the old interface in "The shut-up-about-EnderIO Update." lol

**Comment by HenryLoenwind** [comment]
oh come on, wtf? 2 days after it has been added to Forge people are expecting us to already have rewriten a good portion of our mod for it? we haven't even finished porting the existing features to 1.9.4...

**Comment by tkdberger** [comment]
It's probably just because Forestry did it

**Comment by mezz** [comment]
I added backward compat to forestry, don't worry about it too much.

**Comment by HenryLoenwind** [comment]
sry if I seem a bit annoyed---I've been doing nothing else but porting mods since early February. The finish line for Ender IO is nowhere to be seen and I haven't even **started** on my own primary mod...

**Comment by mezz** [comment]
Believe me I understand. No worries.

**Comment by RealGrep** [comment]
Oh, we understand, believe me. All that work has not gone unnoticed by me and many others. I really can't even start to say how much it's all appreciated. Having EnderIO in 1.9.4 has been absolutely glorious.

And since Forestry and such have added backward compatibility back in, not much of an issue now, anyway. As you say, it's very new functionality, anyway. As for ActuallyAdditions, he was just getting annoyed with users complaining about it, so made a joke out of the release name. Which should, indeed, shut them up about it. :)

tl;dr <3

**Comment by erindalc** [comment]
Take your time Henry. We understand.

**Comment by Brakat07** [comment]
so why the mod owners DO NOT BUT FLUID HANDLER CAPABILITIES WHEN THEY PORT THERE MODS ?
SO ANGRY

**Comment by HenryLoenwind** [comment]
Someone told me once not to respond to idiots, but whatever.

Do you have ANY idea how much time it takes to do that? In you world, making a mod may consist of pressing a button, but that only shows how incredible stupid you are.

I alone put in over 300 hours into porting Ender IO from 1.7.10. Fluid capabilities alone took about 30 hours to get where we are at the moment (about half-way there).

So what do I get from that? I'd say nothing, but that'd be wrong. I get hateful angry people shouting at me.
